tests/integration.rs

```diff
@@ -7,6 +7,156 @@
 #[cfg(feature = "integration_tests")]
 #[macro_use]
 extern crate lazy_static;
+#[macro_use]
+extern crate ctor;
+
+#[ctor]
```

Comment:
There are only specific tests that need these interfaces. I prefer you to move them to the specific test only (i.e. `test_vfio`) and one of the `vhost-user-net` tests.

Reply:
Ended up the vhost_uer_net setups are no longer being used. The rest of network setup commands for VFIO are now only being used by `test_vfio`. PTAL. Thanks!